admin_manual/installation/apps_management_installation.rst

```diff
@@ -42,8 +42,8 @@ page by default.
 
 Click the app name to view a description of the app and any of the app settings in the Application View field.  Clicking the **Enable** button will enable the app.  If the app is not part of the Nextcloud installation, it will be downloaded from the app store, installed and enabled.
 
-Click the gear icon on the lower left to browse experimental apps in the `ownCloud Apps
-Store <https://apps.owncloud.com/>`_. Install experimental apps at your own risk.
+Click the gear icon on the lower left to browse experimental apps in the `Nextcloud App
+Store <https://apps.nextcloud.com/>`_. Install experimental apps at your own risk.
```

Comment:
Needs to be rewritten content-wise. `experimental apps` are currently not shipped over Nextcloud App Store I think. @BernhardPosselt